webpack.config.js

```diff
@@ -57,4 +57,7 @@ module.exports = {
     modules: ['src', 'node_modules'],
   },
   devtool: 'sourcemap',
+  node: {
+    __dirname: false,
+  },
```

Comment:
Nit, let's add a comment so that we don't forget that we absolutely need this to be set to false explicitly here.